install/csm-install/README.md

```diff
@@ -88,6 +89,65 @@ See [Create system configuration](../pre-installation.md#3-create-system-configu
 
 ### 5. Configure management network switches
 
+#### Management network
```

Comment:
Can we move this to its own document and then link here and also from install/common-pre-install/README.md#configure-the-management-network-with-canu

Separately, is this the same set of concerns for upgrades?